### PR TITLE
Add support for Cargo projects in Drafter and in action macros

### PR DESCRIPTION
**Pull request by livingsilver94**

### CODE REVIEW

**Comment by GZGavinZhao** [comment]
Add `%JOBS%`?

**Comment by livingsilver94** [comment]
Oh, right, nice suggestion! Many thanks!

File: source/drafter/build/rust.d [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/* SPDX-License-Identifier: Zlib */
+
+/**
+ * Drafter - Python integration
+ *
+ * Authors: Â© 2020-2022 Serpent OS Developers
```
Comment (ermo) [inline source/drafter/build/rust.d:6]: That Â looks weird...?

**Review by ermo — CHANGES_REQUESTED** [review]
Looks nice.

I assume you have tested it locally to be working?

**Comment by livingsilver94** [comment]
> I assume you have tested it locally to be working?

Still have to. Let me try to package rust into venom to perform a real life test, otherwise should I fail at it i'll perform a simple no-op test.

**Pull request closed without merging**